src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -933,6 +936,37 @@ public static Text of(Object... objects) {
         return builder.build();
     }
 
+    public static Optional<Text> get(Dictionary dictionary, String key, Locale locale) {
```

Comment:
Ideally, developers will pretty much only use `Text.get(String)` to receive their messages. For the most part constant strings should not be used for text creation.